conda-envs/mlos-windows.yml

```diff
@@ -16,7 +16,8 @@ dependencies:
   - jupyter
   - ipykernel
   - nb_conda_kernels
-  - matplotlib
+  - matplotlib<2.9
+  - matplotlib-base<2.9
```

Comment:
Wait, shouldn't this be 3.9?
That's what's specified in `mlos_viz/setup.py`

Reply:
you're right. that was a typo that creeped into the other PR, too